AnkiDroid/src/main/java/com/ichi2/anki/Info.java

```diff
@@ -218,9 +218,20 @@ public String copyDebugInfo() {
             Timber.w(e, "Unable to detect Rust Backend");
         }
 
+        String webviewUserAgent = null;
+        try {
+            webviewUserAgent = getWebviewUserAgent();
+        } catch (Throwable e) {
+            AnkiDroidApp.sendExceptionReport(e, "Info::copyDebugInfo()", "some issue occured while extracting webview user agent");
+            return null;
+        }
 
         String debugInfo = "AnkiDroid Version = " + VersionUtils.getPkgVersionName() + "\n\n" +
                 "Android Version = " + Build.VERSION.RELEASE + "\n\n" +
+                "Manufacturer = " + Build.MANUFACTURER + "\n\n" +
+                "Model = " + Build.MODEL + "\n\n" +
+                "Hardware = " + Build.HARDWARE+"\n\n" +
+                "Webview User Agent = "+ webviewUserAgent +
```

Comment:
```suggestion
                "Webview User Agent = " + webviewUserAgent +
```

spacing nit: https://github.com/ankidroid/Anki-Android/wiki/Code-style#white-space-should-be-used-in-the-following-cases -> "after and before operators"